src/lib.rs

```diff
@@ -224,6 +251,11 @@ impl Url {
         HostType::from(unsafe { ffi::ada_get_host_type(self.0) })
     }
 
+    /// Returns the type of the scheme such as http, https, etc.
+    pub fn scheme(&self) -> SchemeType {
```

Comment:
Let's use a similar syntax like `host_type()`

```suggestion
    pub fn scheme_type(&self) -> SchemeType {
```

Reply:
Updated.